test/Sentry.Testing/VerifyExtensions.cs

```diff
@@ -92,7 +92,7 @@ public override void Write(VerifyJsonWriter writer, DebugImage obj)
             obj.DebugChecksum = ScrubAlphaNum(obj.DebugChecksum);
             obj.DebugFile = ScrubPath(obj.DebugFile);
             obj.CodeFile = ScrubPath(obj.CodeFile);
-            obj.CodeId = ScrubAlphaNum(obj.CodeId);
+            obj.CodeId = obj.CodeId is null ? "null" : "______________";
```

Comment:
```suggestion
            // See https://github.com/getsentry/sentry-dotnet/issues/3467
            // We're replacing the CodeId instead of scrubbing it here to reduce the chance of flake due to different string lengths on different systems
            obj.CodeId = obj.CodeId is null ? "null" : "______________";
```